spring-kafka/src/main/java/org/springframework/kafka/support/KafkaHeaders.java

```diff
@@ -26,7 +26,15 @@
  */
 public abstract class KafkaHeaders {
 
-	private static final String PREFIX = "kafka_";
+	/**
+	 * The prefix for Kafka headers.
+	 */
+	public static final String PREFIX = "kafka_";
+
+	/**
+	 * The prefix for Kafka headers containing 'received' values.
+	 */
+	public static final String RECEIVED = PREFIX + "received";
```

Comment:
Shouldn't we have an underscore in the end of this prefix as well?

Reply:
That would change the header values.

Reply:
Doh. Now I see that this is just a refactoring for existing code without anything semantically new.

Thanks and never mind